GUI/LinuxUtil.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _LINUXUTIL_
+#define _LINUXUTIL_
+
+namespace RTE {
+
+	///<summary>
+	///A utility class with misc static functions for different things, platform specific to
+	///linux
+	///</summary>
+	class LinuxUtil {
+
+//////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
The formatting in this file is still a little wonky. Work with maximdude to get it sorted out please.